common/djangoapps/track/backends/logger.py

```diff
@@ -33,7 +34,10 @@ def __init__(self, name, **kwargs):
         self.event_logger = logging.getLogger(name)
 
     def send(self, event):
-        event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        try:
+            event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        except UnicodeDecodeError:
+            application_log.exception("UnicodeDecodeError Event-Data: %r", event)
```

Comment:
I just checked in with Ed, and he wants us to avoid indiscriminate logging of event data in our app logs.  So let's think about what fields we want to include, and include just those. I think these fields should include:  event.get('event_type'), event.get('event_source'), event.get('page'), and event.get('referer').  Log each, using %r just in case.   Note that any of them might be missing, so get() could return None.  That should have no real PII, but should give us a sense of where the problems are coming from.  (What would be really great would be to find where in the event the actual decode error is coming from, and returning that information.  That's harder, and therefore riskier.  This should just work, and tell us something at least.)